src/main/java/org/folio/oaipmh/client/SourceStorageSourceRecordsClient.java

```diff
@@ -69,8 +67,14 @@ public void postSourceStorageSourceRecords(String idType, Boolean deleted, List
       buffer.appendString(ClientHelpers.pojo2json(List));
     }
 
+
     HttpClientRequest request = this.httpClient.postAbs(this.okapiUrl + "/source-storage/source-records" + queryParams.toString());
     request.handler(responseHandler);
+    request.exceptionHandler(e-> {
+      logger.error("SRS response error 1234: " + e.getMessage(), e);
+      exceptionHandler.handle(e);
+    });
+    request.setTimeout(10000);
```

Comment:
See comment above about using constants instead of integer literals

Reply:
Done